src/OSPSuite.Infrastructure.Import/Core/DataSheetCollection.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using OSPSuite.Utility.Collections;
+
+namespace OSPSuite.Infrastructure.Import.Core
+{
+   /// <summary>
+   ///    a collection of excel dataSheet sheets
+   /// </summary>
+   public class DataSheetCollection : IEnumerable<DataSheet>
+   {
+      private readonly Cache<string, DataSheet> _dataSheets = new Cache<string, DataSheet>(x => x.SheetName);
+
+      public DataSheetCollection (DataSheetCollection dataSheetCollection)
+      {
+         _dataSheets = dataSheetCollection._dataSheets.Copy();
```

Comment:
This might not work. Where is the copy method coming from?

Reply:
I changed this